src/v/cloud_storage/tests/remote_segment_test.cc

```diff
@@ -1152,3 +1154,76 @@ FIXTURE_TEST(test_chunk_prefetch, cloud_storage_fixture) {
         BOOST_REQUIRE(prefetched_chunk.handle.has_value());
     }
 }
+
+FIXTURE_TEST(test_abort_hydration_timeout, cloud_storage_fixture) {
+    config::shard_local_cfg().cloud_storage_hydration_timeout_ms.set_value(0ms);
+
+    auto reset_cfg = ss::defer([] {
```

Comment:
nit: there is the test_utils/scoped_config.h to autoreset a config

Reply:
thanks for pointing this out, changed